plugins/ArchivingMetrics/Clock/Clock.php

```diff
@@ -0,0 +1,16 @@
+<?php
```

Comment:
Some classes are missing their complimentary license comment.

And, unless something breaks, we should try to add `declare(strict_types=1);` to every new class, especially if it is in a new plugin.